### DISCUSSION TITLE
쿠링 Lite에 코드 제안하는 방법 (PR 생성하기)

### DISCUSSION THREAD

**Discussion by x-0o0** [KU-Stacks/kuring-lite-ios#7] [Announcements]
# PR 생성하는 방법

PR은 쿠링 Lite 는 모두가 함께 참여하는 오픈 소스 프로젝트 입니다. PR 생성을 통해 자신의 이름으로 작성된 코드를 쿠링에 들어가는 경험을 해보세요! 참여자들의 깃헙네임이 쿠링앱에 같이 올라갑니다.

## 개요

1. 현재 저장소를 fork 한다. 
2. 작업 내용을 fork 된 저장소에 푸시한다.
3. 본래의 저장소(현재 저장소)에 PR 을 생성한다.

- - -

The process for contributing to a public repository in GitHub starts by *forking the repository*, then *pushing the change* onto the forked, then *creating a pull request* onto the original repository. After that comes the role of the project owner to review and take action (merge/decline) of the requested code change.

**Comment by lgvv** [comment]
이렇게 오픈소스에 기여하는 방법을 배웁니다 !_!🥕
